Stiletto/HeelInfo.cs

```diff
@@ -156,5 +228,93 @@ private void LateUpdate()
                 PostUpdate();
             }
         }
+
+        private void UpdateShoeWarp()
```

Comment:
This is fairly expensive to run on every frame. At the very least check if user changed any of the transform properties and don't run this code if not.

Reply:
Good point, thanks for noticing that. I'll see if I can optimize that a bit.

Reply:
Hopefully better now?

Reply:
Yeah, that works, thanks.